common/djangoapps/nodebb/helpers.py

```diff
@@ -25,7 +28,7 @@ def get_course_related_tabs(request, course):
     course_tabs = get_course_tab_list(request, course)
 
     tabs_dict = collections.OrderedDict()
-    for idx, tab in enumerate(course_tabs):
+    for idx, tab in enumerate(course_tabs):  # pylint: disable=unused-variable
```

Comment:
Can we check if we can eliminate idx as we aren't using this, like if course_tabs is a dictionary we can use course_tabs.values().